flake.nix

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I don't have experience with Nix flakes so probably someone else needs to take a look.